packages/frontend/src/components/CreateUserForm/VerifyEmailForm.tsx

```diff
@@ -3,7 +3,7 @@ import { EmailStatusExpiring } from '@lightdash/common';
 import React, { FC, useEffect } from 'react';
 import Countdown, { zeroPad } from 'react-countdown';
 import { UseFormReturn } from 'react-hook-form';
-import { AnchorLinkWrapper, Subtitle, Title } from '../../pages/SignUp.styles';
+import { AnchorLinkWrapper, Subtitle, Title } from '../../pages/Invite.styles';
```

Comment:
can we get rid of those?

Reply:
yep getting rid of it in the verify email form PR